apps/teamsjs-diagnostic-app/src/components/sample/Scenario1.css

```diff
@@ -0,0 +1,156 @@
+.sr-only {
+  border: 0;
+  clip: rect(0, 0, 0, 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+}
+
+.scenario-container {
+  display: flex;
+  gap: 20px;
+  margin-bottom: 20px;
+}
+
+.scenario1-box,
+.custom-scenario-box,
+.all-api-box {
+  width: 400px;
+  height: 400px;
+  background-color: lightgray;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+  position: relative;
+  overflow: hidden;
+}
+
+.scenario1-button {
+  padding: 10px 20px;
+  font-size: 16px;
+  cursor: pointer;
+  background-color: #464eb8;
+  color: white;
+}
+
+.set-scenario-button {
+  font-size: 16px;
+  cursor: pointer;
+  position: absolute;
+  top: 10px;
+  right: 10px;
+}
+
+.api-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 20px;
+  flex: 1;
+}
+
+.api-header {
+  text-align: center;
+  margin-bottom: 10px;
+  font-weight: bold;
+}
+
+.vertical-box-container {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  overflow-y: auto;
+  max-height: calc(100% - 40px);
+}
+
+.vertical-box {
+  width: 100%;
+  background-color: white;
+  display: flex;
+  align-items: center;
+  border: 1px solid #ccc;
+  padding: 10px;
+  border-radius: 4px;
+}
+
+.box-dropdown {
+  margin-left: auto;
+}
+
+.horizontal-box {
+  width: 100%;
+  height: 600px;
+  overflow: auto;
+  background-color: lightgray;
+}
+
+.content {
+  width: 100%;
+  height: 100%;
+}
+
+.log-console {
+  margin-top: 20px;
+  padding: 20px;
+  background-color: lightgray;
+}
+
+.log-statement {
+  padding: 5px;
+  border-bottom: 1px solid #ccc;
+  overflow: auto;
+}
+
+.box-title {
+  flex-grow: 1;
+}
+
+.box-dropdown {
+  margin-left: 10px;
+}
+
+.all-api-container {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  align-items: flex-start;
+  overflow-y: auto;
+  max-height: 600px;
+  background-color: lightgray;
+  padding: 20px;
+  height: 400px;
+}
+
+.all-api-box {
+  width: 400px;
+  height: 400px;
+  overflow: auto;
+}
+
+.scenario2-header {
```

Comment:
Why is there a scenario 2 css rule in the scenario 1 css file?

Reply:
I will fix the naming! The original structure of my app changed so the naming is now a bit confusing.